pkg/kubectl/cmd/get.go

```diff
@@ -139,6 +139,8 @@ func NewCmdGet(f cmdutil.Factory, out io.Writer, errOut io.Writer) *cobra.Comman
 // RunGet implements the generic Get command
 // TODO: convert all direct flag accessors to a struct and pass that instead of cmd
 func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args []string, options *GetOptions) error {
+	go f.OpenAPISchema(cmdutil.GetOpenAPICacheDir(cmd))
```

Comment:
What  happens to this if the command exits before this is finished?  Should we defer waiting for this to finish or something?